sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicLogicalOperators.scala

```diff
@@ -583,6 +583,16 @@ case class Aggregate(
   }
 }
 
+case class AggregateWithHaving(
+    havingCondition: Expression,
+    child: LogicalPlan)
```

Comment:
`child` always is `Aggregate`, so could we write this like `aggregate: Aggregate`?

Reply:
Thanks, done in c75fe08.